FSharp.Directory.Build.props

```diff
@@ -1,60 +1,6 @@
 <Project>
 
-  <Import Project="build\targets\PackageVersions.props" />
-  <Import Project="build\targets\CommonPackages.targets" />
-
-  <!-- directory locations -->
-  <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
-    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
-    <FSharpSourcesRoot>$(RepoRoot)src</FSharpSourcesRoot>
-    <ToolsRoot>$(RepoRoot)Tools</ToolsRoot>
-    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-    <ProtoOutputPath>$(RepoRoot)Proto\net40\bin</ProtoOutputPath>
-  </PropertyGroup>
-
-  <!-- nuget -->
-  <PropertyGroup>
-    <!-- default NuGet package restore location -->
-    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == ''">$(NUGET_PACKAGES)</NuGetPackageRoot>
-    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == ''">$(MSBuildThisFileDirectory)packages</NuGetPackageRoot>
-    <!-- ensure there is a trailing slash -->
-    <NuGetPackageRoot Condition="!HasTrailingSlash('$(NuGetPackageRoot)') AND '$(OS)' == 'Windows_NT'">$(NuGetPackageRoot)\</NuGetPackageRoot>
-    <NuGetPackageRoot Condition="!HasTrailingSlash('$(NuGetPackageRoot)') AND '$(OS)' != 'Windows_NT'">$(NuGetPackageRoot)/</NuGetPackageRoot>
-  </PropertyGroup>
-
-  <!-- signing -->
-  <PropertyGroup>
-    <SkipSigning>false</SkipSigning>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(FSharpSourcesRoot)\fsharp\msft.pubkey</AssemblyOriginatorKeyFile>
-    <StrongNames>true</StrongNames>
-    <DelaySign>true</DelaySign>
-  </PropertyGroup>
-
-  <!-- mono -->
-  <PropertyGroup>
-    <MonoPackaging Condition="$(TargetFramework.StartsWith('net4')) and '$(OS)' == 'Unix'">true</MonoPackaging>
-  </PropertyGroup>
-
-  <!-- localization -->
-  <PropertyGroup>
-    <EnableXlfLocalization Condition="'$(Configuration)' == 'Proto' or '$(MonoPackaging)' == 'true'">false</EnableXlfLocalization>
-    <UpdateXlfOnBuild Condition="'$(CI)' != '1'">true</UpdateXlfOnBuild>
-  </PropertyGroup>
-
-  <!-- other -->
-  <PropertyGroup>
-    <DebugType>portable</DebugType>
-    <MicroBuildAssemblyFileLanguage>fs</MicroBuildAssemblyFileLanguage>
-    <UseStandardResourceNames>false</UseStandardResourceNames>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-  </PropertyGroup>
-
-  <!-- SDK targets override -->
-  <PropertyGroup Condition="'$(Configuration)' != 'Proto'">
-    <ProtoFSharpTargetsShim>$(ProtoOutputPath)\Microsoft.FSharp.NetSdk.targets</ProtoFSharpTargetsShim>
-    <FSharpTargetsShim Condition="Exists('$(ProtoFSharpTargetsShim)')">$(ProtoFSharpTargetsShim)</FSharpTargetsShim>
-  </PropertyGroup>
+  <Import Project="FSharpBuild.Directory.Build.props" Condition = " '$(UseFSharpBuild)' == '' "/>
```

Comment:
If we're switching on this variable like this, can it be named `$(FSharpTestCompilerVersion)` or similar?  `$(UseFSharpBuild)` isn't immediately obvious to me what it does.